output/mapping/F25_2014.csv

```diff
@@ -34,7 +34,7 @@ xpath,label-key,index,comment,guidance
 /OBJECT_CONTRACT/SHORT_DESCR,descr_short,II.1.4,,Map to `tender.description`
 /OBJECT_CONTRACT/VAL_ESTIMATED_TOTAL,value_excl_vat,,,Map to `tender.value.amount`
 /OBJECT_CONTRACT/VAL_ESTIMATED_TOTAL/@CURRENCY,currency,,,Map to `tender.value.currency`
-/OBJECT_CONTRACT/CALCULATION_METHOD,concession_method_calculate,,,
+/OBJECT_CONTRACT/CALCULATION_METHOD,concession_method_calculate,,,Map to `award.concession.valueCalculationMethod`.
```

Comment:
Section II typically only contains details with respect to the opportunity (i.e. pre-award). I'm not sure why this field is not on F24.

Anyhow, this is about the calculation method for `tender.value`. Therefore, I think this should be `tender.valueCalculationMethod`. (I don't think we need the `.concession` scope, because the semantics for value calculation method don't need to be specific to concessions).

Reply:
I have moved `.valueCalculationMethod` to Tender.